python/modules/DeepTopProducer.py

```diff
@@ -5,6 +5,8 @@
 from itertools import permutations
 import numpy as np
 import itertools
+import os
+import tarfile
```

Comment:
Do we need tarfile module? Can't find the usage below.

Reply:
oh, I removed the usage below, it can be removed